examples/stable_diffusion_v2/train_text_to_image.py

```diff
@@ -152,41 +143,52 @@ def load_pretrained_model_clip_and_vae(pretrained_ckpt, net):
 
 def main(opts):
     dataset, rank_id, device_id, device_num = init_env(opts)
-    LatentDiffusionWithLoss = instantiate_from_config(opts.model_config)
+    latent_diffusion_with_loss = instantiate_from_config(opts.model_config)
     pretrained_ckpt = os.path.join(opts.pretrained_model_path, opts.pretrained_model_file)
-    load_pretrained_model(pretrained_ckpt, LatentDiffusionWithLoss)
-    
+    load_pretrained_model(pretrained_ckpt, latent_diffusion_with_loss)
+
     # lora finetune
     if opts.use_lora:
         # freeze network
-        for param in LatentDiffusionWithLoss.model.get_parameters():
+        for param in latent_diffusion_with_loss.model.get_parameters():
             param.requires_grad = False
 
         # inject lora params
         injected_attns, injected_trainable_params = inject_trainable_lora(
-                                                        LatentDiffusionWithLoss, 
-                                                        use_fp16=(LatentDiffusionWithLoss.model.diffusion_model.dtype==ms.float16),
+                                                        latent_diffusion_with_loss,
+                                                        rank=opts.lora_rank,
+                                                        use_fp16=opts.lora_fp16, #(latent_diffusion_with_loss.model.diffusion_model.dtype==ms.float16),
                                                         )
         assert len(injected_attns)==32, 'Expecting 32 injected attention modules, but got {len(injected_attns)}'
```

Comment:
hardcoded 32 (other places too) may make it hard to adapt lora to other modules (e.g. unet-only for inversion optimisation)